src/command/process_update_task.rs

```diff
@@ -186,7 +186,7 @@ impl UpdateProcessor {
             .get_city_row(&self.chat.selected.clone().unwrap(), number)
             .await?;
 
-        match self.chat.state {
+        match self.chat.before_state {
```

Comment:
nice catch

in my refactoring I forgot to port bot typing also

Reply:
[typing , SleepParams lower](https://github.com/pxp9/weather_bot_rust/pull/23/commits/343ef07a39250a164bfb8386946b90a08ab04494)